.gitlab-ci.yml

```diff
@@ -681,6 +684,53 @@ dogstatsd_arm64_size_test:
   script:
     - inv -e dogstatsd.size-test --skip-build
 
+.build_clang_common:
+  rules:
+    - when: manual
+      allow_failure: true
+  stage: deps_build
+  timeout: 2h 00m
+  script:
+    - wget https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/clang-11.0.0.src.tar.xz -O clang.src.tar.xz
```

Comment:
thoughts on moving some of this to an external script so we can (more) easily run the download job locally.

Reply:
Can you clarify what you mean? Why would we run this locally?